resolwe_bio/processes/workflows/illumina_wes.yml

```diff
@@ -0,0 +1,312 @@
+- slug: workflow-illumina-wes
```

Comment:
It's dubious what Illumina stands for here: is this for Illumina WES kits or for any WES done using Illumina sequencing technologies? In this regard, I am thinking it would be best to use just `worklfow-wes` and than use description field to explain the specifics. Please reflect any changes you make to `slug` also in the `Name` field

Reply:
..and `type` field.